src/Roslyn.Diagnostics.Analyzers/UnitTests/Roslyn.Diagnostics.Analyzers.UnitTests.csproj

```diff
@@ -3,9 +3,10 @@
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <ServerGarbageCollection>true</ServerGarbageCollection>
+    <MicrosoftCodeAnalysisVersion Condition="'$(DotNetBuildFromSource)' != 'true'">$(MicrosoftCodeAnalysisVersionForTests)</MicrosoftCodeAnalysisVersion>
```

Comment:
❔ Are we expecting to build unit tests in source build? Are we expecting those tests to pass when executed? Is anything executing the tests currently?

Reply:
None of the unit tests are executing in source build currently. Should build though.